Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -377,12 +378,17 @@ def _three_factor_corrections_approximation(self, sample_workspace, container_wo
         acc = factor_workspaces['acc']
         acsc = factor_workspaces['acsc']
         assc = factor_workspaces['assc']
-        return (sample_workspace - container_workspace * (acsc / acc)) / assc
+        subtrahend = container_workspace * (acsc / acc)
+        difference = Minus(sample_workspace, subtrahend)
```

Comment:
I just noticed while running a focus with PP enabled that you end up with workspaces with the following names left behind in the ADS:
- subtrahend
- difference

Is it possible to clean these up. Or perhaps there's a way of stopping them being saved into the ADS in the first place?
I guess minuend would also get left behind (see _two_factor function bit lower down)
I think it's fine that the "corrections" workspace group that the CalculatePaalmanPings algorithm creates is left behind but the ones I mentioned above feels more like internal working that should be cleaned up